core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -237,6 +241,10 @@ object ConsoleProducer {
             .withRequiredArg
             .describedAs("producer_prop")
             .ofType(classOf[String])
+    val producerConfigOpt = parser.accepts("producer.config", "Producer config properties file.")
```

Comment:
Just curious, do we need both "producer-property" and "producer.config" for passing the property values? And if yes let's make their naming style consistent.